charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/clusterrole.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if (and (not .Values.customTLSCertificate) (not .Values.certManager.enabled)) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "newrelic-infra-operator.fullname" . }}-admission
```

Comment:
I wonder if we should again apply `truncate 63` when we append to `fullname` or `name` to avoid issues with too long names.